.github/workflows/ci.yml

```diff
@@ -19,62 +19,119 @@ jobs:
         include:
           - operating-system: 'ubuntu-20.04'
             php-version: '7.4'
-            job-description: 'with lowest deps'
+            job-description: 'tests with Symfony ^5'
+            run-tests: 'yes'
+            execute-flex-with-symfony-version: '^5' # explicit check for Symfony 5.x compatibility
+
+          - operating-system: 'ubuntu-20.04'
+            php-version: '7.4'
+            job-description: 'Fixer with lowest deps'
+            run-fixer: 'yes'
             composer-flags: '--prefer-stable --prefer-lowest' # should be checked on the lowest supported PHP version
 
           - operating-system: 'ubuntu-20.04'
             php-version: '7.4'
-            job-description: 'with PHPDoc to type rules'
-            phpdoc-to-type-rules: 'yes' # should be checked on the lowest supported PHP version
+            job-description: 'Fixer with PHPDoc to type rules'
+            run-phpdoc-to-native-type: 'yes' # should be checked on the lowest supported PHP version
 
           - operating-system: 'ubuntu-20.04'
             php-version: '7.4'
-            job-description: 'with Symfony ^5'
+            job-description: 'Fixer with Symfony ^5'
+            run-fixer: 'yes'
             execute-flex-with-symfony-version: '^5' # explicit check for Symfony 5.x compatibility
 
           - operating-system: 'ubuntu-20.04'
             php-version: '8.0'
+            job-description: 'Fixer'
+            run-fixer: 'yes'
+
+          - operating-system: 'ubuntu-20.04'
+            php-version: '8.0'
+            job-description: 'tests'
+            run-tests: 'yes'
 
           - operating-system: 'ubuntu-20.04'
             php-version: '8.1'
-            job-description: 'with Symfony ^6'
+            job-description: 'Fixer with Symfony ^6'
+            run-fixer: 'yes'
+            execute-flex-with-symfony-version: '^6' # explicit check for Symfony 6.x compatibility
+
+          - operating-system: 'ubuntu-20.04'
+            php-version: '8.1'
+            job-description: 'tests with Symfony ^6'
+            run-tests: 'yes'
             execute-flex-with-symfony-version: '^6' # explicit check for Symfony 6.x compatibility
 
+          - operating-system: 'ubuntu-20.04'
+            php-version: '8.2'
+            job-description: 'Fixer'
+            run-fixer: 'yes'
+
           - operating-system: 'ubuntu-20.04'
             php-version: '8.2'
             job-description: 'with migration rules'
-            migration-rules: 'yes' # should be checked on the highest supported PHP version
+            run-fixer: 'yes'
+            run-migration-rules: 'yes' # should be checked on the highest supported PHP version
```

Comment:
we should check both, fixer and tests , when migration rules are applied on highest PHP
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/6656222609/job/18088271502#step:9:1